test/unit/person_test.rb

```diff
@@ -1407,4 +1408,327 @@ def test_updated_not_changed_when_adding_notifiee_info
     assert_equal [project, project2, project3], person.projects.sort_by(&:id)
     assert_equal [project,project2], person.administered_projects.sort_by(&:id)
   end
+
+  test 'merge simple attributes' do
+    person_to_keep = FactoryBot.create(:min_person)
+    other_person = FactoryBot.create(:max_person)
+    other_person_attributes = {}
+    simple_attributes.each do |attribute|
```

Comment:
don't rely on the inherited `simple_attributes` method, but list them explicity

or have a separate test that checks what this method returns

Reply:
Resolved in [8d64fe9](https://github.com/seek4science/seek/pull/1683/commits/8d64fe9b7d6bd86c3eacb81547989a27ec9eb878).